src/Developer/ImageFilter.hxx

```diff
@@ -1,7 +1,6 @@
 #ifndef __itkImageFilter_hxx
 #define __itkImageFilter_hxx
 
-#include "ImageFilter.h"
```

Comment:
Is a similar change needed in the software guide? If there are any instances of this, it is.

Reply:
https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/pull/171 for the coding style guidelines section.